.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,16 @@
 ## Purpose:
 What issue(s) does your PR address?
 
-## Results
-If applicable, briefly report your findings.
+## Strategy
+What was your strategy for this new or edited analysis?
+
+## Concerns/Questions for reviewers:
+What things should reviewers look out for?
 
 ## Pull Request Check List:
-* [ ] Run a linter
-* [ ] Set the seed (if applicable)
-* [ ] Comments and/or documentation up to date
-* [ ] Double check your paths
-* [ ] Check headline numbering
-* [ ] Spell check any Rmd file or md file
-* [ ] Restart R and run all notebooks fresh and save
+* [ ] [Sources are cited](https://github.com/AlexsLemonade/refinebio-examples/CONTRIBUTING.md#citing-sources-in-text)
+* [ ] Set the seed (if applicable).
+* [ ] Remove any manual numbering of sections.
+* [ ] Spell check any Rmd file or md file.
+* [ ] Run [r-lib/styler package](https://github.com/r-lib/styler)
```

Comment:
Did some light googling and it looks like can probably set up GitHub actions to run `styler`: https://github.com/r-lib/actions/tree/master/examples#when-it-can-they-be-used

Reply:
Wow nice find!

Reply:
I would take a look at that repo in general - it seems like there's a bunch of stuff we could think about implementing here! I'll fill an issue about linting specifically.

Reply:
Will do. Interesting!

Reply:
It was pretty easy to just add the styling step before we render. There's now a `--style` option in the render-notebooks.R script. (will add this to the CONTRIBUTING.md docs). So now this can be removed from the checklist.